cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -25,6 +26,12 @@
 def remove_installed_state():
     remove_state('kubernetes-worker.components.installed')
 
+@hook('stop')
+def remove_this_node():
+    ''' Deletes this unit's node when the unit is destroyed. '''
+    kubectl = ['kubectl', '--kubeconfig=/srv/kubernetes/config']
```

Comment:
@chuckbutler Neither this PR nor #48 use manifests, which the `kubectl` function takes as a parameter, so I'm not certain what tack you're suggesting I take here. If the goal is simply to parameterize the location of the kubeconfig, then I'd prefer to abstract the cli thus:

```python

def kubectl(cli param list):
    # call kubectl cli with static kubeconfig defined in this function + cli param list

def kubectl_manifest(blah): #previously kubectl
    # call kubectl function following logic already present
```